src/backend/iohidmanager/device.rs

```diff
@@ -158,6 +170,10 @@ impl Drop for CallbackGuard {
         unsafe {
             IOHIDDeviceRegisterInputReportCallback(self.device.as_concrete_TypeRef(), null_mut(), 0, transmute(null::<()>()), null_mut())
```

Comment:
We should also remove this, Casting null to fn is very bad and it was only suppost to be a quick hack until the function prototype is fixed (https://github.com/jtakakura/io-kit-rs/issues/6), but it seems like this won't happen in the near future, so I think we should just "embed" the correct protoype inline:
```rust
unsafe {
    extern "C" { fn IOHIDDeviceRegisterInputReportCallback(device: IOHIDDeviceRef, report: *mut u8, reportLength: CFIndex, callback: Option<IOHIDReportCallback>, context: *mut c_void); }
    IOHIDDeviceRegisterInputReportCallback(self.device.as_concrete_TypeRef(), null_mut(), 0, None, null_mut())
}
```